frontend/src/components/regions/UsersAndGroups/Presentation.js

```diff
@@ -50,17 +64,15 @@ const TeamSummaryStats = ({ summaryStats }) => {
         <TableRow>
           <TableCell>Pull Requests</TableCell>
           <TableCell>{summaryStats.totalOpenPrs}</TableCell>
-          <TableCell>
-            {oldestOpenPrTime ? oldestOpenPrTime.toLocaleString() : "-"}
+          <TableCell className={prStatusClassName}>
+          {openPrAge ? getAgeString(openPrAge) : "-"}
           </TableCell>
         </TableRow>
         <TableRow>
           <TableCell>Review Cards</TableCell>
           <TableCell>{summaryStats.totalCardsInReview}</TableCell>
-          <TableCell>
-            {oldestCardInReviewTime
-              ? oldestCardInReviewTime.toLocaleString()
-              : "-"}
+          <TableCell className={tildeReviewStatusClassName}>
+          {tildeReviewAge ? getAgeString(tildeReviewAge) : "-"}
```

Comment:
Is this a date or an number? If I ask you your age you would give me a number, not a date. If I ask you your birth date you would give me a date and not a number. 

Rule number 1 of naming things: Call it what it is. 

If I ask you to explain the purpose of tildeReviewAge you would say it's "the date of the oldest review request on Tilde". So a better name might be `dateOfOldestTildeReviewRequest`. It